### DISCUSSION TITLE
xmake是否可以下载子模块

### DISCUSSION THREAD

**Discussion by xiao-mang** [xmake-io/xmake#4230] [Q&A] [answered]
使用xmake的seturl指定url路径，如果该仓库包含一些子模块，会下载吗?

**Comment by waruqi** [comment, marked as answer]
会

**Reply by xiao-mang** [reply]
如果需要下载其归档文件，这种情况下，怎么进行子模块下载呢

**Reply by waruqi** [reply]
你只能设置 git url ，归档文件，github 本身就没支持输出 submodules，你这个要到 github 那去反馈，让他们支持上，而不是这里

**Reply by xiao-mang** [reply]
好哒，谢谢
